apps/chat/src/store/models/models.selectors.ts

```diff
@@ -105,6 +105,10 @@ const selectAllGroupModelKeySet = (state: RootState, references: string[]) => {
   );
 };
 
+const selectCustomModels = createSelector([rootSelector], (state) => {
+  return state.models.filter((model) => model.reference !== model.id);
+});
```

Comment:
It's not good when whole state in dependency
```suggestion
const selectCustomModels = createSelector([_selectModels], (models) => {
  return models.filter((model) => model.reference !== model.id);
});
```
It would be better to create internal selector for `state.models`
```javascript
const _selectModels = (state: RootState) => rootSelector(state).models;
```